### ISSUE TITLE
Incompatibility issue with Bulma 0.4.0

### ISSUE THREAD

**Issue by MBach** [vue-bulma/datepicker#32]
Hi,
With latest release of Bulma (0.4.0), writing forms must be enclosed by DIV with class field, like this:

    <div class="field">
      <label class="label" for="country">Country</label>
      <p class="control is-expanded">
        <span class="select is-fullwidth">
          <select class="is-fullwidth" name="country" v-model="selected_country" @input="updateUser">
            <option v-for="c in countries" v-bind:value="c.id">
              {{ c.country_fr }}
            </option>
          </select>
        </span>
      </p>
    </div>

which produces:
![image](https://cloud.githubusercontent.com/assets/1532657/24142425/c9a3b902-0e26-11e7-863f-c6645afa5f14.png)

If I do the same thing with datepicker and "has-addons" on the enclosing DIV, here's what I have:

    <label class="label">Start date</label>
    <div class="field has-addons">
      <datepicker name="dateForm" v-model="dateForm" :config="{ dateFormat: 'd/m/Y', wrap: true, locale: fr }">
        <a class="button" data-toggle><i class="fa fa-calendar"></i></a>
        <a class="button" data-clear><i class="fa fa-close"></i></a>
      </datepicker>
    </div>

![image](https://cloud.githubusercontent.com/assets/1532657/24142946/697fcd70-0e28-11e7-88ac-856b276c813f.png)

Note that LABEL needs to be outside when using "has-addons". I think this is another issue from Bulma itself.

**Comment by ghost** [comment]
Yeah as a result of this I had to copy the files and edit them to my taste. I'm not git savvy myself but here was my solution:

WrapperInput.vue
```
<template>
    <div class="field has-addons">
        <p class="control is-expanded flatpickr" data-wrap="true" data-clickOpens="false" :class="{ [`has-addons-${alignment}`]: alignment }">
            <input class="input" :class="inputClass" type="text" :placeholder="placeholder" :readonly="readonly" v-model="date" data-input />
        </p>
        <slot></slot>
    </div>
</template>
```

Form.vue (or whatever you call it)
```
<div class="field">
    <div class="control has-icon">
        <datepicker placeholder="Date of Birth" :config="{ dateFormat: 'Y-m-d', wrap: true, maxDate: 'today' }">
            <p class="control">
                <a class="button" data-toggle>
                    <i class="fa fa-calendar"></i>
                </a>
            </p>
            <p class="control">
                <a class="button" data-clear>
                    <i class="fa fa-close"></i>
                </a>
            </p>
        </datepicker>
        <span class="icon is-small">
            <i class="fa fa-birthday-cake"></i>
        </span>
    </div>
</div>
```

I added an icon myself as I'm using it as a date of birth field but of course that is easily omitted.

![image](https://cloud.githubusercontent.com/assets/4530100/24637685/fbb1a5d0-18da-11e7-8058-742f758eb110.png)

**Comment by uptownhr** [comment]
any update here?

**Comment by luventa** [comment]
Sorry for replying so late.
I'm going to upgrade version of Bulma for vue-admin and fix this issue these days.
